lib/project_types/extension/commands/create.rb

```diff
@@ -5,6 +5,9 @@ class Command
     class Create < ShopifyCLI::Command::SubCommand
       prerequisite_task :ensure_authenticated
 
+      recommend_default_node_range
```

Comment:
@vividviolet, do you recall? If not, I can do some testing. I also remember running into NPM issues so we might have to restrict those versions as well.

Reply:
@t6d We are using `import/export` in our build tools which requires Node 12 but the code should be compiled down to `require` statements anyway. The project itself should not require any features from node that I'm aware of. 

As far as NPM issues, I think this went away as we're no longer using `npm list` to check for the renderer version? Can you double check this? If we removed that code then we don't really have a requirement for a specific node version.

I would still suggest a minimum version of Node 10 so that we need to use `fs`. I know we're switching to STDIN but it would cover our bases in case we want to support reading from a config file when building the extension.

Reply:
> As far as NPM issues, I think this went away as we're no longer using npm list to check for the renderer version?

I will double check. I believe this is true for the new code path that uses the new development server. There still seem to be active code paths in the old implementation that utilize npm list to determine the version of the Argo CLI package.

Reply:
Thanks @t6d 🙏🏼. I'll wait then until I have that information to make sure it aligns with extensions' requirements.

Reply:
Given that we're working towards deprecating the code path that still runs `npm|yarn list`, I don't think we should bother introducing a check for `yarn|npm` itself. Feel free to move forward with the PR as is.